automation/build-bin.ts

```diff
@@ -106,7 +108,7 @@ export async function runUnderMsys(argv?: string[]) {
 async function buildPkg() {
 	const args = [
 		'--target',
-		'node10',
+		'host',
```

Comment:
This is the key fix, and it's my bad. In a previous PR (#1328, CLI version 11.4.0), I had specifically changed this option from 'host' to 'node10', aiming at improving consistency among the standalone zip packages (pkg) between Linux / Mac / Windows and Travis / Balena CI, having them all use the same Node 10 version rather than whatever Node was installed, and thus arguably helping with testing (fewer differences to account for). But I misunderstood how it works: `pkg` goes as far as downloading its own patched Node.js interpreter that gets bundled in the final CLI executable, but that interpreter was not used to build the native modules, which were built in the `npm install` step using whatever `node` is available in the path. As it happens, Node 10 was used in the balena CI setup for Linux and Mac, so that change didn't matter on those platforms (and reason why the problem was not detected any sooner, including in my manual testing). But on Windows, the CI setup still uses Node 8 so this led to native modules being compiled with Node 8 but bundled with Node 10 in the standalone package. (Ironically and perversely, one of the main benefits of standalone zip packages is to ensure compatibility between native modules and Node.js, and this bug ensured the very opposite... on Windows, between versions 11.4.0 and 11.9.3.)

To prevent a future regression, I have added a simple `testPkg()` function (below in this same file) that executes `balena version -j` (the built standalone executable) and ensures that the reported Node.js major version matches the major version of `process.version`.